jax/_src/numpy/lax_numpy.py

```diff
@@ -3782,8 +3782,32 @@ def array_equal(a1: ArrayLike, a2: ArrayLike, equal_nan: bool = False) -> Array:
   return reductions.all(eq)
 
 
-@util.implements(np.array_equiv)
 def array_equiv(a1: ArrayLike, a2: ArrayLike) -> Array:
+  """Check if two arrays are shape consistent and element-wise equal.
```

Comment:
I think it's worth expanding the "shape consistent" part here since that's not common usage as far as I know. Perhaps this could say something like "can be broadcasted to the same shape" or similar.